### ISSUE TITLE
Error Phaser 2.4.6 using game.destroy method

### ISSUE THREAD

**Issue by jdnichollsc** [photonstorm/phaser#2387]
Hi guys,

See the error: http://screencast.com/t/cykhh2yVL6
And codepen.io to visualize: http://codepen.io/jdnichollsc/full/MyJbQb/ (click the button and wait for a moment)

Thanks in advance

**Comment by Exilz** [comment]
I have kinda the same error regarding the update method when I delete every children from a group.
I don't know if this is related.

**Comment by semirbabajic** [comment]
+1 here. I'm facing the very same thing with 2.4.6 (don't remember I've seen it with 2.4.4). It would be great to have this fixed (I believe it's something trivial under the hood) as I was planning to switch to the newest release.

Console output is: 
`this.game.time is null`

I didn't run it with the non-minified version so I can't tell the exact line where it snaps, but I believe anyone can replicate it.

Thanks guys.

**Comment by stoneman1** [comment]
Hello,

Its already fixed in the dev branch. Here is the commit :
https://github.com/photonstorm/phaser/commit/4d587d3e40ea515b6c17d226770de235f1c3b378

Basicly just put 
window.removeEventListener('mouseout', this._onMouseOutGlobal, true);
that into mouse.js under stop function

**Comment by mikewesthad** [comment]
@stoneman1 thanks for pointing that out!  I was running into the same issue.  

I'm working on a project where I can't switch to the dev branch and where it makes sense to keep a standard, unmodified version of Phaser 2.4.6.  For anyone in a similar situation, you can reach into the "private" mouse variable and unregister the window event when you destroy the game:

```js
    var mouse = game.input.mouse;
    window.removeEventListener("mouseout", mouse._onMouseOutGlobal, true);
    game.destroy();
```

**Comment by photonstorm** [comment]
Closing this off as it's already fixed in dev, and will hit master in 2.4.7 next week.

**Comment by jdnichollsc** [comment]
Good news! :dancers:
